tests/modules/bclconvert/main.nf

```diff
@@ -0,0 +1,22 @@
+#!/usr/bin/env nextflow
+
+nextflow.enable.dsl = 2
+
+include { BCLCONVERT } from '../../../modules/bclconvert/main.nf'
+
+process STUB_BCLCONVERT_INPUT {
+    output:
+    path "SampleSheet.csv"          ,emit: samplesheet
+    path "DDMMYY_SERIAL_FLOWCELL"   ,emit: run_dir
+
+    stub:
+    """
+    mkdir DDMMYY_SERIAL_FLOWCELL
+    echo "SampleSheet" > SampleSheet.csv
+    """
+}
```

Comment:
no need to fake the process, we can just fake the input

Reply:
How do you mean? 
I'd be happy to be able to run a real world test, but I since `bcl-convert` is very particular about it's inputs and the structure of it, I think it would be hard to support this. Not to mention the size of the data needed. Even a very small MiSeq run is a couple of Gb.